docs/DeveloperGuide.md

```diff
@@ -1055,22 +1055,176 @@ testers are expected to do more *exploratory* testing.
+   
+#### Deleting a log
+
+1. Deleting a log from a friend while all friend are being shown
+
+    1. Prerequisites: List all persons using the `listfriend` command. Multiple friends in the list, log must be present to delete
+
+    2. Test case: `deletelog n/Alex Row id/1`<br>
+       Expected: Alex Row's first log is deleted
+
+    3. Test case: `deletelog 1 id/1`<br>
+       Expected: The first log of the friend who is at index 1 is deleted
+
+    4. Other incorrect delete commands to try: `deletelog 1 n/Alex Row id/1`, `deletelog 1 id/x`, `...` (where x is larger than the list size)<br>
+       Expected: Error message provided for incorrect format or, incorrect index provided.
+
+#### Editing a log
+
+1. Editing a friend's log while all friends are being shown
+    1. Prerequisites: Atleast one friend with a log is present.
+
+    2. Test case: `editlog n/Alex Yeoh id/1 ttl/New Log Title` <br>
+       Expected: `Alex Yeoh`'s first log's title is changed to `New Log description`
+
+    3. Test case: `editfriend 2 id/1 d/The new description` <br>
+       Expected: First log's description of friend at index `2` in the friend list is edited to have a description `New Log description`
+    
+### Event management
+
+#### Adding an event
+
+1. Adding an event while all events are being shown
+    1. Test case: `addevent n/Old Event dt/14-04-2020 1600` <br>
+       Expected: No change in the GUI as the event is past event, although if `listevent -a` is run this event should be visable.
+    2. Test case: `addevent n/Future Event dt/14-04-2023 1300` <br>
+       Expected: GUI should be updated with a new event called `Future Event`
+    3. Test case: `addfriend n/Date Check dt/99-04-2023 1240` <br>
+       Expected: Error message should be displayed concerning the incorrect format of date.
+
+2. Adding an event while in another tab
+    1. Prerequisites: Must be in either `Friends` or `Insights` tab.
+
+    2. Test case: `addevent n/Party dt/14-04-2023 d/Cool party` <br>
+       Expected: Tab should switch to `Events` tab and a new event `Party` should be present in the events list.
+
+#### Deleting an event
+
+1. Deleting an event while all events are being shown
+
+    1. Prerequisites: List all events using the `listevents` command. Multiple events in the list.
+
+    2. Test case: `deletevent 1`<br>
+       Expected: First event is deleted from the list. Details of the deleted event shown in the status message.
+
+    3. Test case: `deletefriend 0`<br>
+       Expected: No event is deleted. Error details shown in the status message. Status bar remains the same.
+
+    4. Other incorrect delete commands to try: `deleteevent`, `deleteevent x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+#### Editing an event
+
+1. Editing an event while all events are being shown
+
+    1. Prerequisites: Atleast 3 events present, with one their names being `Birthday`
```

Comment:
Why must one of the event names be birthday

Reply:
This is so that atleast one result is displayed, when the user tests.